### DISCUSSION TITLE
Trying to run hub server in container on Google Compute Engine

### DISCUSSION THREAD

**Discussion by marians** [jacktrip/jacktrip#1308] [General]
Hi, I'm trying to run the `jacktrip/jacktrip:2.3` docker image as a container on Google Compute Engine. So far, I cannot connect to the server via jacktrip, but can't figure out why.

When I run that image locally on my machine, things work fine and I see the `jackd` and the `jacktrip` process:

```
USER       PID %CPU %MEM    VSZ   RSS TTY      STAT START   TIME COMMAND
root         1  2.2  0.2  21112 11008 ?        Ss   06:37   0:00 /sbin/init
root        12  0.2  0.0   4452  3584 pts/0    Ss   06:37   0:00 /bin/sh
root        23  0.5  0.2  26560  9088 ?        Ss   06:37   0:00 /usr/lib/systemd/systemd-journald
jack        31 11.0  3.1 194660 124800 ?       S<Lsl 06:37   0:00 /usr/local/bin/jackd -d dummy -C 0 -P 0 --rate 48000 --period 128
jacktrip    39  4.0  0.3  21780 14544 ?        S<s  06:37   0:00 /usr/local/bin/jacktrip -S -t -z --hubpatch 4 --bufstrategy 4 -q auto
```

However on my GCE instance, I only see this `jack_wait` process:

```
USER         PID %CPU %MEM    VSZ   RSS TTY      STAT START   TIME COMMAND
root           1  0.1  0.2  21116 11976 ?        Ss   06:25   0:00 /sbin/init
root          19  0.1  0.2  26560 10176 ?        Ss   06:25   0:00 /usr/lib/systemd/systemd-journald
jacktrip     199  0.5  0.1  72456  4348 ?        S<s  06:29   0:00 /usr/local/bin/jack_wait -w -t 5
```

Any idea what's missing?

**Comment by marians** [comment]
Meanwhile I found out that I can access the logs in the container using `journalctl`. Here is an informative part:

```
Jun 07 06:47:30 jacktrip-b128 jack[32]: Cannot lock down 105039644 byte memory area (Cannot allocate memory)
Jun 07 06:47:31 jacktrip-b128 systemd[1]: jack.service: Main process exited, code=dumped, status=7/BUS
Jun 07 06:47:31 jacktrip-b128 systemd[1]: jack.service: Failed with result 'core-dump'.
```

**Comment by mikedickey** [comment]
Are you using docker or podman, and what is the command line you used to start it? You may need to update limits to allow for the amount of locked memory required.

**Comment by marians** [comment]
Thanks for the reply. Since it was Google Compute launching the container in some way, I didn't execute any specific command. Anyway, I gave up with jacktrip. Closing the issue.
